Readability.js

```diff
@@ -102,7 +102,7 @@ Readability.prototype = {
   DEFAULT_MAX_PAGES: 5,
 
   // Element tags to score by default.
-  DEFAULT_TAGS_TO_SCORE: ["SECTION", "P", "TD", "PRE"],
+  DEFAULT_TAGS_TO_SCORE: "section,h2,h3,h4,h5,h6,p,td,pre".toUpperCase().split(","),
```

Comment:
It feels a bit suspicious to me that this fixes the issue of images not appearing. Is this just a coincidence because these images are siblings of some of these header elements? Should we instead be increasing the content score for elements that hold images?

I'm not too worried about landing since, since it doesn't regress the other test cases and does improve this case, but it just feels like it might be a coincidental fix.